src/core/drive/form_submission.js

```diff
@@ -251,7 +217,7 @@ export class FormSubmission {
   }
 }
 
-function buildFormData(formElement: HTMLFormElement, submitter?: HTMLElement): FormData {
+function buildFormData(formElement, submitter) {
```

Comment:
How would one express that? The variable now needs to be renamed?

Reply:
@pailhead The right way to rewrite the function signature is clearly `buildFormData(formElement, submitterWhichIsAnOptionalHTMLElement)`.

Reply:
Did you already forget the only correct, Hungarian Notation? `mhtmle_submitter`, where `m` is for `maybe`, to mimic Haskell style, `html` namespaces the complex types present in the browser and `e` is for element of course.

/s

Reply:
Oh wow, I didn’t know this could be expressed so clearly. Thank you for clarifying.

Reply:
This one is interesting. I would have no idea what a "submitter's" purpose is here with or without Typescript.

I am excited to see how this code base changes without Typescript. The "types" or what things are will have to be communicated in some way. This may lead to very readable code.